pkgs/tools/games/minecraft/packwiz/default.nix

```
@@ -0,0 +1,37 @@
+{ lib
+, buildGo118Module
+, fetchFromGitHub
+, installShellFiles
+}:
+
+buildGo118Module rec {
+  pname = "packwiz";
+  version = "unstable-2022-5-25";
+
+  src = fetchFromGitHub {
+    owner = "packwiz";
+    repo = "packwiz";
+    rev = "e71b63ea98283c8c1f0e03ee51ae40f452f22a61";
+    sha256 = "sha256-XwGacEVfQAduDCSMQFRw7Xnx4bND2zaV7l27B+2u5xg=";
+  };
+
+  vendorSha256 = "sha256-M9u7N4IrL0B4pPRQwQG5TlMaGT++w3ZKHZ0RdxEHPKk=";
+
+  nativeBuildInputs = [
+    installShellFiles
+  ];
+
+  postInstall = ''
+    installShellCompletion --cmd packwiz \
+      --bash <($out/bin/packwiz completion bash) \
+      --fish <($out/bin/packwiz completion fish) \
+      --zsh <($out/bin/packwiz completion zsh)
+  '';
+
+  meta = with lib; {
+    description = "A command line tool for editing and distributing Minecraft modpacks, using a git-friendly TOML format";
+    homepage = "https://packwiz.infra.link/";
+    license = licenses.mit;
+    maintainers = with maintainers; [ infinidoge ];
```

```suggestion
    maintainers = with maintainers; [ infinidoge ];
    mainProgram = "packwiz";
```
Because of the unstable in the version which messes with `nix run`

---

Weird, it runs just fine with `nix run` from my testing. Isn't it supposed to grab from `pname`?

---

(Tested with `nix run github:Infinidoge/nixpkgs/new/packwiz#packwiz`, for what it's worth)

---

Though I can't reproduce, added since it is of no cost.

---

You're right.. in my 5 months of inactivity something must have changed... Oh well